eval/api_runner.py

```diff
@@ -214,20 +266,33 @@ def run_api_eval(args):
         output_dir = os.path.dirname(output_file)
         if not os.path.exists(output_dir):
             os.makedirs(output_dir)
+
+        with open(output_file.replace(".csv", ".json"), "w") as f:
+            json.dump(output_df.to_dict("records"), f)
+
         try:
             output_df.to_csv(output_file, index=False, float_format="%.2f")
         except:
             output_df.to_pickle(output_file)
 
-        results = output_df.to_dict("records")
+        if logprobs:
+            print(f"Writing logprobs to JSON file at {output_file.replace('.csv', '.json')}")
+            results = output_df.to_dict("records")
+            with open(
```

Comment:
nit: shall we add a os.path.exists check here and exit/fail early with an error message to clone the repo if it hasn't been setup yet?

Reply:
Good point, fixed!